src/util/mapbox.js

```diff
@@ -365,11 +383,7 @@ class TelemetryEvent {
 
         const payload: Object = {
             event: this.type,
-            created: new Date(timestamp).toISOString(),
-            sdkIdentifier: 'mapbox-gl-js',
-            sdkVersion,
-            skuId: SKU_ID,
-            userId: this.anonId
```

Comment:
Removing these params from the payload would also remove it from the `MapSessionAPI` class because it extends the `TelemetryEvent` class but does not overload the `postEvent` method.

Reply:
Added more unit tests to check the entire payload in https://github.com/mapbox/mapbox-gl-js/commit/af215cf90ce754460ccfd84c86148f40b69569ce.

Reply:
I thought we were also sending this payload in `MapSessionAPI`, but I just checked that, and we don't do that.